src/sentry/static/sentry/app/views/settings/organizationMembers/organizationAccessRequests.jsx

```diff
@@ -92,4 +130,8 @@ class OrganizationAccessRequests extends React.Component {
   }
 }
 
-export default OrganizationAccessRequests;
+const StyledButton = styled(Button)`
+  margin-right: ${space(1)};
+`;
```

Comment:
Would <3 you forever if you could remove Flex and Box while you were in here. But no biggie if not